src/source/Common/Lws/LwsCall.c

```diff
@@ -73,6 +73,10 @@ STATUS blockingLwsCall(PRequestInfo pRequestInfo, PCallInfo pCallInfo)
 
 CleanUp:
 
+    if(retStatus == STATUS_IOT_CREATE_LWS_CONEXT_FAILED)
```

Comment:
Are you targeting this particular status code only? Perhaps, adding

`CHK_LOG_ERR(retStatus);`